embedded-graphics/README.md

```diff
@@ -11,188 +11,250 @@
+* [ssd1331](https://crates.io/crates/ssd1331): SPI (4 wire) driver for the SSD1331 OLED display
+* [ssd1351](https://crates.io/crates/ssd1351): SSD1351 driver
+* [ssd1675](https://crates.io/crates/ssd1675): Rust driver for the Solomon Systech SSD1675 e-Paper display (EPD) controller
+* [st7735-lcd](https://crates.io/crates/st7735-lcd): Rust library for displays using the ST7735 driver
+* [st7789](https://crates.io/crates/st7789): A Rust driver library for ST7789 displays
+* [st7920](https://crates.io/crates/st7920): ST7920 LCD driver in Rust
 
-[![Collage of drawing examples](https://raw.githubusercontent.com/jamwaffles/embedded-graphics/master/doc/assets/all_drawing_ops.png)](https://github.com/jamwaffles/embedded-graphics/blob/master/doc/drawing-examples.md)
+## Simulator
 
-Example usage of drawing primitives, text and images with embedded-graphics can be found [here](https://github.com/jamwaffles/embedded-graphics/blob/master/doc/drawing-examples.md).
+Embedded graphics comes with a [simulator]!
 
-## Getting help/reporting an issue
+![It can display all sorts of embedded-graphics test code.](https://raw.githubusercontent.com/jamwaffles/embedded-graphics/master/assets/simulator-demo.png)
 
-If you think you've found a bug, or would like to suggest a new feature to add to embedded-graphics, please [open an issue](https://github.com/jamwaffles/embedded-graphics/issues/new).
+Take a look at the [simulator examples] to see what
+embedded-graphics can do, and how it might look on a display. You can run the examples like
+this:
 
-If you need more deeper/more personalized help, please check out the [embedded-graphics Matrix channel](https://matrix.to/#/#rust-embedded-graphics:matrix.org).
+```bash
+git clone https://github.com/jamwaffles/embedded-graphics.git
+cd embedded-graphics
 
-## Additional functions provided by external crates
+cargo run -p embedded-graphics-simulator --example hello
+```
 
-- [BMP images - `tinybmp`](https://crates.io/crates/tinybmp)
-- [TGA images - `tinytga`](https://crates.io/crates/tinytga)
-- [ProFont monospace font - `profont`](https://crates.io/crates/profont)
-- [Picofont Pico8 font - `embedded-picofont`](https://crates.io/crates/embedded_picofont)
-- [IBM437 font - `ibm437`](https://crates.io/crates/ibm437)
-- [Simple layout/alignment functions - `embedded-layout`](https://crates.io/crates/embedded-layout)
+[simulator]: https://github.com/jamwaffles/embedded-graphics/tree/c4f74c12dae9f0a0193fa48192f905a002bf8c9d/simulator
+[simulator examples]: https://github.com/jamwaffles/embedded-graphics/tree/c4f74c12dae9f0a0193fa48192f905a002bf8c9d/simulator/examples
 
-Note that some of these crates may not support the latest version of embedded-graphics.
+## Crate features
 
-If you know of a crate that is not in this list, please [open an
-issue](https://github.com/jamwaffles/embedded-graphics/issues/new).
+Add these to your `Cargo.toml` to turn on extra bits of functionality.
```

Comment:
IMO this should be more generic, because I wouldn't consider `fixed_point` an "extra bit of functionality".